src/wallet/crypter.h

```diff
@@ -5,12 +5,14 @@
 #ifndef BITCOIN_WALLET_CRYPTER_H
 #define BITCOIN_WALLET_CRYPTER_H
 
-#include <keystore.h>
 #include <serialize.h>
 #include <support/allocators/secure.h>
+#include <script/signingprovider.h>
 
 #include <atomic>
 
+#include <boost/signals2/signal.hpp>
```

Comment:
This should probably move to wallet.h as well, since `NotifyStatusChanged` moved there.

Reply:
Fixed